cmd/terramate/e2etests/cloud/run_cloud_sync_preview_test.go

```diff
@@ -176,6 +214,8 @@ func TestCLIRunWithCloudSyncPreview(t *testing.T) {
 
 			s := sandbox.New(t)
 
+			err := os.Unsetenv("GITHUB_ACTIONS")
```

Comment:
This environment variable is already removed from the CLI environ, check the `NewCLI()` implementation. We never remove env vars from the test process, we just filter out and dont pass them down to the Terramate binary.
Also, calling `os.Unsetenv()` unsets it globally, so does not make sense to run on each test case.